atlasdb-coordination-impl/src/main/java/com/palantir/atlasdb/coordination/ValueAndBound.java

```diff
@@ -28,6 +30,8 @@
  * valid at that timestamp (i.e. the bound is greater than or equal to that timestamp).
  */
 @Value.Immutable
+@JsonSerialize(as = ImmutableValueAndBound.class)
```

Comment:
How did this ever work if we needed to add it?

Reply:
the internal backup product had a workaround that just copied this class :-/